CHANGELOG.md

```diff
@@ -237,6 +237,7 @@
 * [BUGFIX] `mimirtool rules prepare`: do not add aggregation label to `on()` clause if already present in `group_left()` or `group_right()`. #7839
 * [BUGFIX] Analyze Grafana: fix parsing queries with variables. #8062
 * [BUGFIX] `mimirtool rules sync`: detect a change when the `query_offset` or the deprecated `evaluation_delay` configuration changes. #8297
+* [CHANGE] Analyze Rules: Count recording rules used in rules group as used. #6133
```

Comment:
move this line at just after the last `[CHANGE]` entry
cf: https://github.com/grafana/mimir/tree/main/docs/internal/contributing#changelog